simplistic_editor/README.md

```diff
@@ -4,81 +4,85 @@ and contract styled ranges of text. For more information visit https://api.flutt
 
 # Structure
 ## Visualization Layer
-The layer that showcases the TextEditingDelta history in a ListView.
+The layer that showcases the `TextEditingDelta` history in a ListView. These widgets are all unique to this sample
+and are unlikely to contain much logic the typical user will need.
```

Comment:
Added this sentence:

  > These widgets are all unique to this sample and are unlikely to contain much logic the typical user will need.

Can you confirm that this is accurate, @Renzo-Olivares?

Reply:
Accurate!